package.json

```diff
@@ -1,115 +1,115 @@
-    "plotly.js": "2.x",
-    "react": "18.x",
-    "react-i18next": "11.x",
-    "react-router-dom": "6.x",
-    "rxjs": "6.x"
-  },
-  "devDependencies": {
-    "@openmrs/esm-extensions": "next",
-    "@openmrs/esm-framework": "next",
-    "@openmrs/esm-react-utils": "next",
-    "@openmrs/esm-styleguide": "next",
-    "@swc/cli": "^0.1.62",
-    "@swc/core": "^1.3.62",
-    "@swc/jest": "^0.2.26",
-    "@testing-library/dom": "^8.20.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^14.4.3",
-    "@types/jest": "^28.1.8",
-    "@types/react": "^18.2.8",
-    "@types/react-dom": "^18.2.4",
-    "@types/react-router": "^5.1.20",
-    "@types/react-router-dom": "^5.3.3",
-    "@types/webpack-env": "^1.18.1",
-    "@typescript-eslint/parser": "^5.59.9",
-    "carbon-components-react": "^8.34.0",
-    "concurrently": "^7.6.0",
-    "css-loader": "^6.8.1",
-    "dayjs": "^1.11.9",
-    "eslint": "^8.42.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-config-ts-react-important-stuff": "^3.0.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "file-saver": "^2.0.5",
-    "husky": "^8.0.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^28.1.3",
-    "jest-cli": "^28.1.3",
-    "jest-environment-jsdom": "^28.1.3",
-    "lerna": "^5.6.1",
-    "openmrs": "next",
-    "plotly.js": "^2.24.3",
-    "prettier": "^2.8.8",
-    "pretty-quick": "^3.1.3",
-    "raw-loader": "^4.0.2",
-    "react": "^18.2.0",
-    "react-csv": "^2.2.2",
-    "react-dom": "^18.2.0",
-    "react-grid-layout": "^1.4.1",
-    "react-i18next": "^11.18.6",
-    "react-pivottable": "^0.11.0",
-    "react-plotly.js": "^2.0.0",
-    "react-router-dom": "^6.11.2",
-    "react-table": "^7.8.0",
-    "rxjs": "^6.6.7",
-    "swc-loader": "^0.2.3",
-    "turbo": "^1.10.12",
-    "typescript": "^4.9.5",
-    "webpack": "^5.85.1",
-    "webpack-cli": "^5.1.3"
-  },
-  "packageManager": "yarn@3.6.3"
+    "name": "@ugandaemr/esm-data-visualizer-app",
+    "version": "1.0.0",
+    "license": "MPL-2.0",
+    "description": "A microfrontend for reporting in UgandaEMR",
+    "browser": "dist/esm-ugandaemr-data-visualizer-app.js",
+    "main": "src/index.ts",
+    "source": true,
+    "scripts": {
+        "start": "openmrs develop --backend http://194.163.171.253:8282",
+        "serve": "webpack serve --mode=development",
+        "build": "webpack --mode production",
+        "analyze": "webpack --mode=production --env analyze=true",
+        "lint": "TIMING=1 eslint src --ext js,jsx,ts,tsx",
+        "prettier": "prettier --write \"src/**/*.{ts,tsx}\"",
+        "typescript": "tsc",
+        "test": "jest --config jest.config.js --passWithNoTests",
+        "verify": "turbo lint typescript test",
+        "coverage": "yarn test -- --coverage",
+        "prepare": "husky install"
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "pretty-quick --staged && yarn verify"
+        }
+    },
```

Comment:
@Daphne210 You created a new file or just changed few lines

Reply:
i think @Daphne210  changed a few lines.